packages/wallet-connectors/src/BrowserWallet.ts

```diff
@@ -122,7 +122,7 @@ export class BrowserWalletConnector implements WalletConnector, WalletConnection
         return this.client.sendTransaction(accountAddress, type, payload);
     }
 
-    async signMessage(accountAddress: string, message: string): Promise<AccountTransactionSignature> {
+    async signMessage(accountAddress: string, message: string | SignMessageObject): Promise<AccountTransactionSignature> {
```

Comment:
The schema is the schema that should be used to deserialize the message data, and we usually encode them with base64.
The data is the message data, as hex because we usually encode stuff other than schemas as hex :thinking:

Reply:
Is this the same kind of schema used to serialize parameters? Used for the wallet to display what you sign decoded to a string? What if you want to sign arbitrary bytes that don't look like a parameter?

Is there a reason for `data` not being a `Buffer`?

Reply:
I don't think we'll allow arbitrary bytes that cannot be displayed using a schema.

The BW API does not include buffers, seeing that they are not a "native" type in a web environment.